draft-hodges-webauthn-registries.xml

```diff
@@ -153,10 +153,8 @@ https://github.com/w3c/webauthn/pulls?q=is%3Apr+label%3Aspec%3Awebauthn-registri
 	RFC Editor - please delete this request after the registries have been created. ]]
       </t>
       <t>
-        For both registries, the expert(s) and IANA will interact as follows.
-        IANA will direct any incoming requests regarding either of these registries to
-        this document and, if defined, the processes established
-        by the expert(s); typically, this will mean referring them to the registry Web page.
+	Any additional processes established by the expert(s) after the publication of this document
+	will be recorded on the registry Web page at the experts'/expert's discretion.
```

Comment:
did you intend..
```suggestion
	will be recorded on the registry Web page at the experts' discretion.
```
..?

Reply:
This was Amanda's proposed language - exactly.  Ultimately the RFC Editors will make it say it the way they want it to say it.  But I thought it was strange too.

I can make the change you suggest, or perhaps "expert(s)'".  Let me know your preference and I'll make the edits right away.

Reply:
oh gosh, do whatever you think is best at this time and yeah the RFC Editor/Editors will do their thing in any case... ;-)